spark/src/test/scala/org/apache/mahout/sparkbindings/drm/DrmLikeSuite.scala

```diff
@@ -63,6 +65,65 @@ class DrmLikeSuite extends FunSuite with DistributedSparkSuite with DrmLikeSuite
           throw new AssertionError("Block must be dense.")
         keys -> block
     }).norm should be < 1e-4
+
+  }
+
+  test("DRM wrap labeled points") {
+
+    import org.apache.spark.mllib.linalg.{Vectors => SparkVector}
+    import org.apache.spark.mllib.regression.LabeledPoint
+
+    val sc = mahoutCtx.asInstanceOf[SparkDistributedContext].sc
```

Comment:
A `MahoutDistributedContext` should be implicitly converted into a SparkDistributedContext when it is needed.  This could be written as : 

     ``` val sc:SparkDistributedContext```